torch/_torch_docs.py

```diff
@@ -8365,6 +8370,11 @@ def merge_dicts(*dicts):
 Note:
     Equivalent to input[:,::-1]. Requires the array to be at least 2-D.
```

Comment:
I'm now noticing these notes. I think both the fliplr and flipud suggestions for advanced in indexing will actually throw errors in PyTorch? Would you check? If so, seems like we should remove these notes?

Reply:
You are correct indeed.

```python
>>> import torch
>>> t = torch.ones((3,2))
>>> t[:, ::-1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: step must be greater than zero
>>> t[:, ::-1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: step must be greater than zero
>>> t[::-1, ...]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: step must be greater than zero
>>> t[::1, ...]
tensor([[1., 1.],
        [1., 1.],
        [1., 1.]])
```

Reply:
Updating it to
```
Note:
    Requires the array to be at least 2-D.
```

Reply:
On second look should we make it,
```
Note:
    Requires the tensor to be at least 2-D.
```

Reply:
That would be more appropriate. Sure.